app/main.py

```diff
@@ -1,8 +1,21 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
The `people` dictionary is a class variable, which means it is shared across all instances of the `Person` class. This could lead to unexpected behavior if multiple instances are created with the same name, as they will overwrite each other in the dictionary.